CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required (VERSION 3.2)
+project (benchmark C)
+
+################################################################################
+# Packages
+################################################################################
+find_package(Threads REQUIRED)
+if(NOT THREADS_FOUND)
+	message(FATAL_ERROR "Threads not found")
+endif()
+
+include(GNUInstallDirs)
+
+################################################################################
+# Compiler flags:
+#   We want to use the same flags in the entire optee_client git
+################################################################################
+add_compile_options (
+	-Wall -Wextra -Werror
+#	-Wall -Wbad-function-cast -Wcast-align
```

Comment:
I guess it's commented on purpose because of actual warnings in `benchmark`?

Reply:
Yes, we can fix that later on.